mythril/laser/ethereum/natives.py

```diff
@@ -53,30 +56,35 @@ def ecrecover(data):
     return o
 
 
-def sha256(data):
+def sha256(data: Union[bytes, str]) -> bytes:
     try:
         data = bytes(data)
     except TypeError:
         raise NativeContractException
     return hashlib.sha256(data).digest()
 
 
-def ripemd160(data):
+def ripemd160(data: Union[bytes, str]) -> bytes:
     try:
         data = bytes(data)
     except TypeError:
         raise NativeContractException
-    return 12 * [0] + [i for i in hashlib.new("ripemd160", data).digest()]
+    digest = hashlib.new("ripemd160", data).digest()
+    padded = 12 * [0] + list(digest)
+    return bytes(padded)
 
 
-def identity(data):
+def identity(data: Union[bytes, str]) -> bytes:
```

Comment:
I think the input type here will be same as that of starting_calldata which is a list

Reply:
I think this would be in conflict with the other signatures for the native crypto functions, because they're all called in a row [here](https://github.com/ConsenSys/mythril-classic/blob/feature/type-hints-develop/mythril/laser/ethereum/natives.py#L90) with the same input for each